### ISSUE TITLE
VisuallyHidden styles sometimes aren't visually hidden!

### ISSUE THREAD

**Issue by sarahmonster** [octopusthink/nautilus#222]
Because our VisuallyHidden styles have a height and width of 1px, in some cases elements can actually be visible as a tiny little coloured dot on the page.

It seems like we need that 1px size for elements to be read out, so maybe the clip properties just need to be fixed?

Related reading:

https://snook.ca/archives/html_and_css/hiding-content-for-accessibility

**Comment by sarahmonster** [comment]
Looks like it should be as follows:

```
clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
  clip: rect(1px, 1px, 1px, 1px);
```

But when I tried that, the SkipLink doesn't seem to appear on focus, so we'll need to adjust more.

https://github.com/octopusthink/getmicdrop.com/commit/725093ca6dc3f0c3d6e219dd6d45a83c9488c867
